bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java

```diff
@@ -878,8 +878,10 @@ private boolean activateRule(final WrappedRule rule) {
         if (slTriggers.stream()
                 .anyMatch(t -> ((BigDecimal) t.getConfiguration().get(SystemTriggerHandler.CFG_STARTLEVEL))
                         .intValue() <= startLevelService.getStartLevel())) {
-            runNow(rule.getUID(), true, Map.of(SystemTriggerHandler.OUT_STARTLEVEL, StartLevelService.STARTLEVEL_RULES,
-                    "event", SystemEventFactory.createStartlevelEvent(StartLevelService.STARTLEVEL_RULES)));
+            runNow(rule.getUID(), true,
+                    Map.of(SystemTriggerHandler.OUT_STARTLEVEL, StartLevelService.STARTLEVEL_RULES, "event",
+                            SystemEventFactory.createStartlevelEvent(StartLevelService.STARTLEVEL_RULES,
+                                    RuleEngineImpl.class.getName() + ".activateRule")));
```

Comment:
event sources should generally be the package name, not the class name. so just org.openhab.core.automation here...